x11-themes/faenza-icon-theme/faenza-icon-theme-1.3.1-r1.ebuild

```diff
@@ -34,6 +32,7 @@ src_prepare() {
 		cp "${x}"/places/scalable/start-here-gentoo.svg \
 			"${x}"/places/scalable/start-here.svg || die
 	done
+	default
```

Comment:
`xdg_src_prepare` is already calling `default`, don't do it twice.

Reply:
done